packages/@webex/plugin-meetings/test/unit/spec/roap/turnDiscovery.ts

```diff
@@ -70,7 +70,8 @@ describe('TurnDiscovery', () => {
       mediaId: expectedMediaId,
       audioMuted: testMeeting.isAudioMuted(),
       videoMuted: testMeeting.isVideoMuted(),
-      meetingId: testMeeting.id
+      meetingId: testMeeting.id,
+      preferTranscoding: true
```

Comment:
it would be nice to have a test confirming that we set preferTranscoding: false for multistream meetings, basically the same test like existing "sends TURN_DISCOVERY_REQUEST, waits for response and sends OK" but with the testMeeting.isMultistream = true

Reply:
done .